### PR TITLE
Adds CSV Formatting option to Org Search

### PR DESCRIPTION
**Pull request by kpitzen**
# Description

This is an extension to https://github.com/pulumi/pulumi.ai/issues/126 - CSV output was requested as a number of users would likely prefer to do aggregate work on something that can easily be imported into excel / google sheets etc.

Fixes https://github.com/pulumi/pulumi.ai/issues/126

## Checklist

- [x] I have run `make tidy` to update any new dependencies
- [x] I have run `make lint` to verify my code passes the lint check
  - [ ] I have formatted my code using `gofumpt`

- [ ] I have added tests that prove my fix is effective or that my feature works

- [x] I have run `make changelog` and committed the `changelog/pending/<file>` documenting my change

- [ ] Yes, there are changes in this PR that warrants bumping the Pulumi Cloud API version

### CODE REVIEW

File: pkg/cmd/pulumi/org_search.go [hunk 1]
```diff
@@ -31,11 +32,70 @@ import (
...
+func (d *Delimiter) Set(v string) error {
+	if v == "\\t" {
+		*d = Delimiter('\t')
+	} else if len(v) != 1 {
+		return errors.New("delimiter must be a single character")
+	} else {
+		*d = Delimiter(v[0])
+	}
+	return nil
+}
```
Comment (kpitzen) [inline pkg/cmd/pulumi/org_search.go:54]: This is hideous, and I'm surprised I couldn't find a better way to map characters to runes - any input would be much appreciated!

Comment (kpitzen) [reply pkg/cmd/pulumi/org_search.go:54]: (As it is, it allows for single-character delimiters and tabs, which should cover the vast majority of use cases, but it'd be great to be able to support more, if needed)

File: pkg/cmd/pulumi/org_search.go [hunk 2]
```diff
@@ -170,7 +200,11 @@ func newSearchCmd() *cobra.Command {
 	)
 	cmd.PersistentFlags().VarP(
 		&scmd.outputFormat, "output", "o",
-		"Output format. Supported formats are 'table', 'json', and 'yaml'.",
+		"Output format. Supported formats are 'table', 'json', 'csv', and 'yaml'.",
+	)
+	cmd.PersistentFlags().Var(
```
Comment (justinvp) [inline pkg/cmd/pulumi/org_search.go:205]: Should there be a default? Like a `,` (comma)?

**Review by justinvp — APPROVED** [review]
LGTM otherwise

File: pkg/cmd/pulumi/org_search.go (hunk 2 above)
Comment (kpitzen) [reply pkg/cmd/pulumi/org_search.go:205]: There is - when one is not provided, comma is used by default.

**Pull request merged**